core/app/lib/uffizzi_core/concerns/models/account.rb

```diff
@@ -27,21 +27,26 @@ module UffizziCore::Concerns::Models::Account
     has_many :payments, dependent: :destroy
 
     aasm(:state) do
-      state :active, initial: true
+      state :trial, initial: true
+      state :require_card
+      state :active
       state :payment_issue
       state :disabled
-      state :draft
 
       event :activate do
-        transitions from: [:payment_issue, :disabled, :draft], to: :active
+        transitions from: [:payment_issue, :disabled, :require_card, :trial], to: :active
+      end
+
+      event :enable_paywall do
```

Comment:
Maybe `enable_paywall ` => `disable_trial` or `require_card`

Name of state should be a noun I think. 
Maybe for state `require_card` => `card_request` because `require` is a verb